src/Seq.Input.HealthCheck/HealthCheckInput.cs

```diff
@@ -34,6 +34,26 @@ public class HealthCheckInput : SeqApp, IPublishJson, IDisposable
             InputType = SettingInputType.LongText)]
         public string TargetUrl { get; set; }
 
+        [SeqAppSetting(InputType = SettingInputType.Password, IsOptional = true, DisplayName = "Authentication Header",
+            HelpText = "An optional `Name: Value` header, stored as sensitive data, for authentication purposes.")]
+        public string AuthenticationHeader { get; set; }
+        
+        [SeqAppSetting(InputType = SettingInputType.LongText,
+            IsOptional = true,
+            DisplayName = "Additional Header",
+            HelpText = "Optional headers sent to each URL in format of `Attribute: Value`.\n" + 
+                       "Headers always need a value, attributes cannot contain spaces, duplicate attributes will be " +
+                       "overwritten in order of appearance.")]
+        public string AdditionalHeader { get; set; }
```

Comment:
Would prefer to make name (`OtherHeaders`), documentation, and behavior identical to _Seq.App.HttpRequest_ here, if we can.